x-pack/plugins/security_solution/public/timelines/containers/api.test.ts

```diff
@@ -506,3 +508,143 @@ describe('persistTimeline', () => {
     });
   });
 });
+
+describe('importTimelines', () => {
+  const fileToImport = { fileToImport: {} } as ImportDataProps;
+  const fetchMock = jest.fn();
+
+  beforeAll(() => {
+    jest.resetAllMocks();
```

Comment:
the jest resets seem repetitive, happening twice in beforeAll and afterAll... is that necessary? same thing in each describe. can we cut these down?

Reply:
maybe a `beforeEach` would eliminate the need within the describes? idk, give it a try

Reply:
yeah, I think keep reset in just beforeAll would be enough